scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -36,31 +36,30 @@ public class CompileTargets
             "vbc.exe"
         };
 
-        public static readonly string[] HostPackageSupportedRids = new[]
-        {
-            "win7-x64",
-            "win7-x86",
-            "osx.10.10-x64",
-            "osx.10.11-x64",
-            "ubuntu.14.04-x64",
-            "centos.7-x64",
-            "rhel.7-x64",
-            "rhel.7.2-x64",
-            "debian.8-x64"
-        };
+        public static string HostPackagePlatformRid => HostPackageSupportedRids[
+                             (PlatformServices.Default.Runtime.OperatingSystemPlatform == Platform.Windows)
+                             ? $"win7-{PlatformServices.Default.Runtime.RuntimeArchitecture}"
+                             : PlatformServices.Default.Runtime.GetRuntimeIdentifier()];
 
-        public static readonly string[] HostPackages = new[]
+        public static readonly Dictionary<string, string> HostPackageSupportedRids = new Dictionary<string, string>()
```

Comment:
It is used above in the lambda `HostPackagePlatformRid` which is used in more places.